### ISSUE TITLE
 Support for the 'new' dotnet target framework monikers

### ISSUE THREAD

**Issue by Tasteful** [xunit/xunit#683]
Make xUnit to understand both the old and new target framework monikers, see https://github.com/aspnet/Announcements/issues/98

**Comment by bradwilson** [comment]
Those TFMs have been changed. This has the updated information: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md

**Comment by Tasteful** [comment]
Is the tfm from the announcement not valid?

**Comment by bradwilson** [comment]
> Is the tfm from the announcement not valid?

That's correct. They changed their mind on the TFM name.

**Comment by justsayno** [comment]
I agree. I have now realised that to reference a class library (IE the library under test) from my xUnit project I must add both

``
"frameworks": {
    "net451": { },
    "dnx451": { }
  }
``

Or Visual studio complains about it (still builds but tests don't get found *most* of the time). Very weird inconsistent behaviour. But after much headache I noticed if I add both like above to the target referenced library it works. I don't understand enough about the frameworks and what is going on to really know if this is an xUnit problem or a Visual Studio 2015 problem....

Is there any difference in how the test will run the class library in question regarding the fact that presumably the reason it stops complaining about not being about to find the referenced class is because it is running under the `dnx451` and not the newly named `net451`?
